### DISCUSSION TITLE
[Feature] Upload single file from mobile app

### DISCUSSION THREAD

**Discussion by Vyerni** [immich-app/immich#3080] [Feature Request]
### The feature

Rather than having full backup selected and uploading the entire camera album, and given you can scroll through the Immich app and see the items in your gallery that are not yet on your server. It would be nice to be able to select individual files that you wish to back up.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1684

**Comment by Vyerni** [comment]
Apologies, I searched on phone but obviously incorrectly.
